### PR TITLE
ks version update to 0.11.0, and bug fixes.

### PR DESCRIPTION
**Pull request by kunmingg**
Now both ksonnet app and go library have version 0.11.0.
Bug fixes:
1. create namespace for create-app request if not already exists.
2. read RegUri from know-registries config if not exist in create-app request.
3. API change following version upgrade.

fix https://github.com/kubeflow/kubeflow/issues/1340

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1356)

### CODE REVIEW

File: bootstrap/bootstrapper.yaml [hunk 1]
```diff
@@ -52,17 +52,17 @@ spec:
...
       - name: kubeflow-bootstrapper
-        image: gcr.io/kubeflow-images-public/bootstrapper:v20180618-715aafc
+        image: gcr.io/kubeflow-images-public/bootstrapper-minikube:v20180813-cbc1c06-dirty-b82a7f
         workingDir: /opt/bootstrap
         command: [ "/opt/kubeflow/bootstrapper"]
         args: [
           "--in-cluster",
           "--namespace=kubeflow",
-          "--apply",
+          "--apply"
```
Comment (richardsliu) [inline bootstrap/bootstrapper.yaml]: Looks like the comma was removed by accident?

**Comment by kunmingg** [comment]
@jlewi 
Will remove baked registries in following PRs.

File: bootstrap/bootstrapper.yaml (hunk 1 above)
Comment (kunmingg) [reply bootstrap/bootstrapper.yaml]: The following line is commented out so remove this one as well

**Pull request merged**